src/kube-runtime/src/plugins/plugin_utils.py

```diff
@@ -16,36 +16,41 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-from __future__ import print_function
-
 import logging
 import argparse
 import yaml
 
-logger = logging.getLogger(__name__)
+from common.utils import init_logger
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PluginHelper:  #pylint: disable=too-few-public-methods
+    def __init__(self, plugin_config: dict):
+        self._parameters = plugin_config.get("parameters", "")
+        self._plugin_name = plugin_config.get("plugin", "")
+        self._failure_policy = plugin_config.get("failurePolicy", "fail")
```

Comment:
is there a validation for this field?

Reply:
If this field is not `ignore`, we will treat it as `fail`.  Refer to https://github.com/microsoft/pai/pull/4048/files#diff-33c702285a3718b9aae88340cbe60538R38. Not validate it here, just trust rest-server will pass valid config to runtime.
Invalid config may fail the runtime and job.